src/main/java/org/datadog/jmxfetch/util/CustomLogger.java

```diff
@@ -41,9 +42,9 @@ public static void setup(Level level, String logLocation) {
     }
 
     /** Laconic logging for reduced verbosity. */
-    public static void laconic(Logger logger, Level level, String message, int max) {
+    public static void laconic(org.slf4j.Logger logger, String message, int max) {
```

Comment:
I'm pretty sure you're going to hate this suggestion, but I'm going to throw this out there just in case. What do you think about using reflection to call the right method depending on `Level level`? That way we could keep log level support for the laconic logging.

Reply:
I decided to use a switch statement instead.